pkg/upgrade/types.go

```diff
@@ -27,10 +28,12 @@ type Upgrader interface {
 }
 
 type simpleUpgrader struct {
-	pluginConfig   api.PluginConfig
-	accountsClient azureclient.AccountsClient
-	storageClient  storage.Client
-	kubeclient     kubernetes.Interface
+	pluginConfig                    api.PluginConfig
+	accountsClient                  azureclient.AccountsClient
+	storageClient                   storage.Client
+	virtualMachineScaleSetVMsClient azureclient.VirtualMachineScaleSetVMsClient
+	virtualMachineScaleSetsClient   azureclient.VirtualMachineScaleSetsClient
+	adminKubeClient                 kubernetes.Interface
```

Comment:
`s/adminKubeClient/kubeClient`

We have an issue about stop using admin kubeconfig during upgrades, no need to name the client field after it tho here.

Reply:
ok